src/main/resources/uk/gov/hmrc/scalatestaccessibilitylinter/accessibility-linter.conf

```diff
@@ -242,7 +242,7 @@ known-issues = [ # same format as hmrc/page-accessibility-check
   {
     tool = "axe"
     descriptionRegex = """Fix any of the following:[\n\r\s]+Some page content is not contained by landmarks"""
-    snippetRegex = """<a href="#" class="govuk-back-link">Back</a>"""
+    snippetRegex = """<a href=".*" class="govuk-back-link">Back</a>"""
```

Comment:
not sure if we should limit this to `[^"]+`

Reply:
@oscarduignan in other places I think we're using `.*` for links URLs etc., wanna chat about this one?

Reply:
I don't mind sticking with current convention then doing a more general review if it's ever flagged we want to reduce time it takes to run the accessibility tests

Reply:
(because probably regex isn't adding much, and we'd get more benefit in terms of speed from other optimization, and with the different things we're checking it's unlikely this accidentally applies to a issue we didn't want it to)